WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailLikesView.swift

```diff
@@ -98,11 +98,11 @@ private extension ReaderDetailLikesView {
         case (true, 0):
             summaryLabel.attributedText = highlightedText(SummaryLabelFormats.onlySelf)
         case (true, 1):
-            summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.singularWithSelf, totalLikes))
+            summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.singularWithSelf))
         case (true, _) where totalLikes > 1:
             summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.pluralWithSelf, totalLikes))
         case (false, 1):
-            summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.singular, totalLikes))
+            summaryLabel.attributedText = highlightedText(String(format: SummaryLabelFormats.singular))
```

Comment:
(Similar to the above comment.)

```suggestion
            summaryLabel.attributedText = highlightedText(SummaryLabelFormats.singular)
```